modules/create-policy-from-system-policies-view.adoc

```diff
@@ -8,58 +8,67 @@
 You can create new security policies from the system policies view.
 
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Policies*.
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Policy Management*.
 . Click *Create policy*.
 . Enter the following details about your policy in the *Policy details* section:
 ** Enter a *Name* for the policy.
 ** Optional: Attach notifiers to the policy by selecting from the available *Notifiers* under the *Attach notifiers* section.
 +
 [NOTE]
 ====
-You must integrate {product-title} with your notification provider, for example, webhooks, Jira, PagerDuty, Splunk, or others before you can forward alerts.
+You must integrate {product-title-short} with your notification provider, for example, webhooks, Jira, PagerDuty, Splunk, or others before you can forward alerts.
```

Comment:
```suggestion
Before you can forward alerts, you must integrate {product-title-short} with your notification provider, such as webhooks, Jira, PagerDuty, Splunk, or others.
```
This is my attempt at reducing the number of commas here for better readability.